ca/ca.go

```diff
@@ -415,7 +414,7 @@ func (ca *CertificateAuthorityImpl) IssueCertificate(ctx context.Context, csr x5
 	serialBytes[0] = byte(ca.prefix)
 	_, err = rand.Read(serialBytes[1:])
 	if err != nil {
-		err = core.InternalServerError(err.Error())
+		err = berrors.InternalServerError("boulder/ca.IssueCertificate: failed to generate serial: %s", err)
```

Comment:
So I just realized the reason I did this was to differentiate errors from the `go-grpc/grpc` package from ones from our `boulder/grpc` package. Looking at it now I think it probably doesn't actually matter since we include the method name on the end and the other package just uses the prefix `grpc:`.